templates/front/docs_faq.html

```diff
@@ -0,0 +1,58 @@
+{% extends "front/base_docs.html" %}
+{% load staticfiles %}
+
+{% block title %}FAQ - healthchecks.io{% endblock %}
+
+{% block docs_content %}
+<div class="faq">
+<h2>Frequently Asked Questions</h2>
+<h2>General</h2>
+<a name="What is Healthchecks?"></a>
+<h2 class="rule">What is Healthchecks?</h2>
```

Comment:
FAQs shouldn't be hardcoded.
Create another model or database table for the FAQs.
It should be possible to have them dynamically updated.